Project.toml

```diff
@@ -2,22 +2,20 @@ name = "JLSO"
 uuid = "9da8a3cd-07a3-59c0-a743-3fdc52c30d11"
 license = "MIT"
 authors = ["Invenia Technical Computing Corperation"]
-version = "2.6.0"
+version = "2.7.0"
 
 [deps]
 BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
 CodecZlib = "944b1d66-785c-5afd-91f1-9de20f533193"
 FilePathsBase = "48062228-2e41-5def-b9a4-89aafe57970f"
-Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 
 [compat]
 BSON = "0.2.4, 0.3"
 CodecZlib = "0.6, 0.7"
 FilePathsBase = "0.7, 0.8, 0.9"
-Memento = "0.10, 0.11, 0.12, 0.13, 1"
-TimeZones = "0.9, 0.10, 0.11, 1"
+TimeZones = "0.9, 0.10, 0.11, ~1.0, ~1.1, ~1.2, ~1.3, ~1.4, ~1.5"
```

Comment:
This is unrelated to this PR.
CI has actually been broken for ages.
https://github.com/invenia/JLSO.jl/actions/runs/1428133005

This is because we have specimen files that were saved with TimeZones prior to v1.5.